recipe/meta.yaml

```diff
@@ -1,23 +1,22 @@
 {% set name = "pandas" %}
-{% set version="0.23.0rc1" %}
+{% set version = "0.23.0rc1" %}
```

Comment:
We actually need this to the bot can parse the version and send PRs updating the package. I'll check if the `master` branch needs this change too.

Reply:
Thanks. If it’s easier I was just planning to merge this to master, even though it’s just an RC. Will that cause issues?

Reply:
Yeah, I just didn't want to mess anything up by trying to do that myself :) To get a `dev` branch, should I

1. Branch `dev off the current master
2. Cherry pick https://github.com/conda-forge/pandas-feedstock/pull/41/commits/7bd7d85edbdbd8bdcee851d3daf68bd1315e797c
3. Push that branch to upstream

Is there any way to skip the CI builds when I push upstream? Will `[ci skip]` do it?